src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+<html lang='en-US'>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Leaderboard</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <nav>
+            <div>
+                <h1 class='title'>Leaderboard</h1>
+            </div>
+        </nav>
+        <div class='main-container d-flex justify-content-center'>
+            <div class='left-container'>
+                <div class='title-container d-flex'>
+                    <h3 class='scores-title'>Recent Scores</h3>
+                    <button id='reload' class='refresh-btn' type='button'>Refresh</button>
+                </div>
+                <div class='scores-table border'>
+                    <ul class='scores-list'>
+                        <!-- Insert Scores here -->
+                    </ul>
+                </div>
+            </div>
+            <div class='right-container d-flex justify-content-center'>
+                <h3 class='add-score'>Add your score</h3>
+                <div class='input-container'>
+                    <input id='name' class='input-box border' type='text' placeholder="Your name">
+                    <br>
+                    <input id='score' class='input-box border' type='text' placeholder="Your score">
```

Comment:
[Optional]

- [ ] It will be better if you have input validation. Currently, I can submit anything as a score.